tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AppProvider>
+    <Page title="Judge.Me Reviews">
+      <Layout>
+        <LayoutSection>
+          <Button @click="toggleCollapsibleActive" style="margin-bottom: 20px;">{{ formButtonMessage }}</Button>
+          <Collapsible :open="collapsibleActive" :class="{ collapsibleActive: collapsibleActive }">
+            <Card>
+              <ReviewForm @add-review="addReview" :key="formReset" />
+            </Card>
+          </Collapsible>
+          <Checkbox label="Unverified reviews only" v-model="unverifiedOnly" />
+          <Divider />
+          <EmptyState
+            v-if="!reviews || reviews.length < 1"
+            heading="No reviews found">
+            <Text as="p">Add a review above or reload the page</Text>
+          </EmptyState>
+          <ReviewCard
+            v-for="(review, index) in reviews"
+            :key="review.id"
+            :reviewObject="review"
+            :unverifiedOnly="unverifiedOnly"
+            @remove-review="removeReview(review.id, index)"
```

Comment:
Nit: I would use init `reviews` as an object {...}, and use keys to access it, rather than use indexes to access an array